seacrowd/sea_datasets/dengue_filipino/dengue_filipino.py

```diff
@@ -0,0 +1,132 @@
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@INPROCEEDINGS{8459963,
+      author={E. D. {Livelo} and C. {Cheng}},
+      booktitle={2018 IEEE International Conference on Agents (ICA)},
+      title={Intelligent Dengue Infoveillance Using Gated Recurrent Neural Learning and Cross-Label Frequencies},
+      year={2018},
+      volume={},
+      number={},
+      pages={2-7},
+      doi={10.1109/AGENTS.2018.8459963}}
+    }
+"""
+
+_LANGUAGES = ["fil", "tgl"]
+
+_DATASETNAME = "dengue_filipino"
+
+_DESCRIPTION = """\
+Benchmark dataset for low-resource multi-label classification, with 4,015 training, 500 testing, and 500 validation examples, each labeled as part of five classes. Each sample can be a part of multiple classes. Collected as tweets.
+"""
+
+_HOMEPAGE = "https://github.com/jcblaisecruz02/Filipino-Text-Benchmarks"
+
+_LICENSE = Licenses.GPL_3_0.value
```

Comment:
is it expected to be a mistake or do you find in other places suggesting this dataset license is a GPL? bcs I find this dataset has unknown license

Reply:
The huggingface code linked [this](https://huggingface.co/datasets/dengue_filipino/blob/d81e2d1cbcf6049ecb324a872d4e3aac60b87743/dengue_filipino.py#L40) as its home page, and it has [GPL license](https://github.com/jcblaisecruz02/Filipino-Text-Benchmarks?tab=GPL-3.0-1-ov-file#readme)